### PR TITLE
change card order at K's request and fix sponsor ranking

### PR DESCRIPTION
**Pull request by schneider-clarisse**
sponsor sort change due to Safari voodoo

### CODE REVIEW

File: src/js/pages/attendee/Home.jsx [hunk 1]
```diff
@@ -132,7 +132,7 @@ export default class Featured extends React.Component {
 
   setSponsors = () => {
     const sponsors = Array.from(BrandStore.sponsors);
-    sponsors.sort((a, b) => a.rank > b.rank);
+    sponsors.sort((a, b) => a.rank - b.rank);
```
Comment (larajanecka) [inline src/js/pages/attendee/Home.jsx:135]: why this change?

**Review by larajanecka — APPROVED** [review]

File: src/js/pages/attendee/Home.jsx (hunk 1 above)
Comment (schneider-clarisse) [reply src/js/pages/attendee/Home.jsx:135]: safari fucks this up for some reason
